develop/components/pages/AccordionPage.tsx

```diff
@@ -39,20 +50,36 @@ const AccordionPage: React.FC = () => {
 
                     <p>Alternative theme</p>
                     <div className="result wide">
-                        <Accordion list={lists[3]} alternative />
+                        <Accordion alternative defaultValue={3}>
+                            {lists[3].map((item: AccordionItemProps, i: number) => {
+                                return <AccordionItem key={i} {...item} />;
+                            })}
+                            <AccordionItem
+                                header="Accordion header"
+                                subHeader={
+                                    <span>
+                                        Expanded by default when setting <Code>defaultValue</Code> to <Code>3</Code> (index) in the parent <Code>Accordion</Code>
+                                    </span>
+                                }
+                            >
+                                {lists[3][2].children}
+                            </AccordionItem>
+                        </Accordion>
                     </div>
                 </div>
             </div>
         </div>
     );
 };
 
-function generateList(): Array<AccrodionListItem> {
+const Code: React.FC = (props) => <code className="d-inline">{props.children}</code>;
+
+function generateList(): Array<AccordionItemProps> {
     return [
         {
             header: loremIpsum({ units: "words", count: 3 }),
             subHeader: loremIpsum({ units: "words", count: 5 }),
-            content: (
+            children: (
```

Comment:
children makes it sound like it should be an array. maybe you should have left it as content 🤣

Reply:
I only removed content, you can render like this:
```tsx
<AccordionItem>Some content</AccordionItem>
```
Which is what the majority of users might prefer. However, if you want to generate a list yourself, you can always use the predefined `children` prop.